src/main/kotlin/org/jetbrains/research/tasktracker/ui/main/panel/MainPluginPanelFactory.kt

```diff
@@ -5,40 +5,59 @@ import com.intellij.openapi.wm.ToolWindow
 import com.intellij.openapi.wm.ToolWindowFactory
 import com.intellij.ui.components.JBPanel
 import com.intellij.ui.jcef.JBCefApp
-import java.awt.GridLayout
+import com.intellij.util.ui.JBUI
+import org.jetbrains.research.tasktracker.ui.main.panel.template.IndexPageTemplate
+import org.jetbrains.research.tasktracker.ui.main.panel.template.TasksPageTemplate
+import java.awt.BorderLayout
+import java.awt.FlowLayout
 import java.awt.event.ActionListener
 import javax.swing.JButton
 
 class MainPluginPanelFactory : ToolWindowFactory {
-    private val button = JButton("click me")
+    private val nextButton = buttonNonPaintedBorder("next")
+    private val backButton = buttonNonPaintedBorder("back")
+    private lateinit var mainWindow: MainPluginWindow
 
     override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
-        val mainWindow = project.getService(MainWindowService::class.java).mainWindow
+        mainWindow = project.getService(MainWindowService::class.java).mainWindow
         val jComponent = toolWindow.component
-        val panel = JBPanel<JBPanel<*>>(GridLayout(2, 1)).apply { // TODO need we GridLayout?
+        backButton.isVisible = false
+        nextButton.addListener {
+            selectTask()
+        }
+        mainWindow.jComponent.size = JBUI.size(toolWindow.component.width, toolWindow.component.height)
+        val buttonPanel = JBPanel<JBPanel<*>>(FlowLayout()).apply {
+            add(backButton)
+            add(nextButton)
+        }
+        val panel = JBPanel<JBPanel<*>>(BorderLayout()).apply {
             add(mainWindow.jComponent)
-            add(
-                JBPanel<JBPanel<*>>().apply {
-                    add(
-                        button.apply {
-                            addListener {
-                                mainWindow.loadDefaultPage("tasks")
-                                // TODO refactor, add function for button listener and text change
-                            }
-                        }
-                    )
-                }
-            )
+            add(buttonPanel, BorderLayout.SOUTH)
         }
-        jComponent.parent.add(panel)
+        jComponent.add(panel)
     }
 
     override fun isApplicable(project: Project) = super.isApplicable(project) && JBCefApp.isSupported()
 
+    private fun selectTask() {
+        mainWindow.loadDefaultPage(TasksPageTemplate)
+        nextButton.text = "select"
```

Comment:
I'd prefer if you will store all these textx in properties, e.g. like [this](https://github.com/JetBrains-Research/refactoring-workshop-demo/blob/main/demo-plugin/src/main/resources/messages/DemoPluginBundle.properties)